resources/teamcity/web/keyman-web-release.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright (C) 2025 SIL International. All rights reserved.
+# Distributed under the MIT License. See LICENSE.md file in the project
+# root for full license information.
+#
+# TC build script to build release of KeymanWeb.
+
+# shellcheck disable=SC2164
+# shellcheck disable=SC1091
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# shellcheck disable=SC2154
+. "${KEYMAN_ROOT}/resources/teamcity/includes/tc-helpers.inc.sh"
+
+################################ Main script ################################
+
+builder_describe \
+  "Run tests for native KeymanWeb" \
+  "all            run all actions" \
+  "build          build Web + embedded" \
+  "publish        publish release"
+
+builder_parse "$@"
+
+cd "${KEYMAN_ROOT}/web"
+
+function build_web_action() {
+  builder_echo start web_build "Building KeymanWeb"
+
+  node ../resources/gosh/gosh.js ./ci.sh build
+
+  builder_echo end web_build success "Finished building KeymanWeb"
+}
+
+function _push_release_to_skeymancom() {
+  # Push release to s.keyman.com/kmw/engine (do this before updating
+  # downloads.keyman.com so we can ensure files are available)
+  builder_echo start publish "Publishing release to skeyman.com"
+
+  cd ../../s.keyman.com
+  git pull https://github.com/keymanapp/s.keyman.com.git master
+  cd ../keyman/web
+  node ../resources/gosh/gosh.js ./ci.sh prepare:s.keyman.com --s.keyman.com ../../s.keyman.com
+
+  builder_echo end publish success "Finished publishing release to skeyman.com"
+}
+
+function _zip_and_upload_artifacts() {
+  if ! is_windows; then
+    # requires Powershell
+    return 0
+  fi
+
+  builder_echo start "zip and upload artifacts" "Zipping and uploading artifacts"
+
+  cd "${KEYMAN_ROOT}/resources/teamcity/web"
+  powershell -NonInteractive -ExecutionPolicy Bypass -File zip-and-upload-artifacts.ps1
+  cd "${KEYMAN_ROOT}/web"
+
+  builder_echo end "zip and upload artifacts" success "Finished zipping and uploading artifacts"
+}
+
+function _upload_help() {
+  builder_echo start "upload help" "Uploading new Keyman for Web help to help.keyman.com"
+
+  cd "${KEYMAN_ROOT}/resources/build"
+  node ../gosh/gosh.js ./help-keyman-com.sh web
```

Comment:
```suggestion
  ./help-keyman-com.sh web
```

Ditto on help.keyman.com path dependency that is implicit here, has a req on env var HELP_KEYMAN_COM which we should set here and pass as a param from TC.